deploy/helm/charts/templates/contoller.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "zfslocalpv.fullname" . }}-controller
+  {{- with .Values.controller.annotations }}
+  annotations: {{ toYaml . | nindent 4 }}
+  {{- end }}
+  labels:
+    {{- include "zfslocalpv.controller.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "zfslocalpv.controller.matchLabels" . | nindent 6 }}
+  serviceName: {{ .Values.serviceAccount.controller.name }}
```

Comment:
serviceName in the operator yaml is "openebs-zfs", here we are using service account name as service name. I hope this is not an issue cc: @prateekpandey14